### PR TITLE
Use 3D 64-bit positions

### PR DESCRIPTION
**Pull request by Pessimistress**
For #3831 

#### Change List
- All `vec2 *64xyLow` attributes are now `vec3 *64Low`
- Documentation

### CODE REVIEW

File: modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js [hunk 1]
```diff
@@ -150,8 +150,6 @@ export default class GPUGridAggregator {
       // cache weights and position data to process when data is not changed
       weights: null,
       gridPositions: null,
-      positionsBuffer: null,
-      positions64xyLowBuffer: null,
```
Comment (Pessimistress) [inline modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js]: @1chandu I'm pretty sure these are no longer used. They are still mentioned in the GPUGridAggregator documentation though.

Comment (1chandu) [reply modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js]: Yes, ok to remove,  code related to building/using these buffers is already removed in #3777.

File: docs/shader-modules/project64.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Projected position in the clipspace.
 
 ```glsl
 void project_position_fp64(vec4 position_fp64, out vec2 out_val[2])
-void project_position_fp64(vec2 position, vec2 position64xyLow, out vec2 out_val[2])
+void project_position_fp64(vec2 position, vec2 position64Low, out vec2 out_val[2])
```
Comment (1chandu) [inline docs/shader-modules/project64.md]: Nit: `vec2 position64Low` -> `vec3 position64Low`

Comment (Pessimistress) [reply docs/shader-modules/project64.md]: `project_position_fp64` only deals with x and y.

**Pull request merged**